plugins/modules/nmcli.py

```diff
@@ -70,7 +70,7 @@
             - If you want to control non-ethernet connection attached to V(bond), V(bridge), or V(team) consider using O(slave_type) option.
```

Comment:
Can you add that `loopback` is added in community.general 8.1.0?

Reply:
Added